python/paddle/fluid/incubate/fleet/parameter_server/pslib/__init__.py

```diff
@@ -263,6 +263,19 @@ def shrink_dense_table(self, decay, scope=None, table_id=None):
                                                    decay)
         self._role_maker._barrier_worker()
 
+    def clear_model(self):
+        """
+        clear_model() will be called by user. It will clear sparse model.
+
+        Example:
+            >>> fleet.clear_model()
```

Comment:
using ".. code-block:: python" may be better

Reply:
done